index.html

```diff
@@ -10581,7 +10582,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p>Indicates the current "checked" <a>state</a> of checkboxes, radio buttons, and other <a>widgets</a>. See related <sref>aria-pressed</sref> and <sref>aria-selected</sref>.</p>
 				<p>The <sref>aria-checked</sref> <a>attribute</a> indicates whether the <a>element</a> is checked (<code>true</code>), unchecked (<code>false</code>), or represents a group of other elements that have a mixture of checked and unchecked <span>values</span> (<code>mixed</code>). Most inputs only support values of <code>true</code> and <code>false</code>, but the <code>mixed</code> value is supported by certain tri-state inputs such as a <rref>checkbox</rref> or <rref>menuitemcheckbox</rref>.</p>
 				<p>The <code>mixed</code> value is <em>not</em> supported on <rref>radio</rref>, <rref>menuitemradio</rref>, <rref>switch</rref> or any element that inherits from these, and <a>user agents</a> MUST treat a <code>mixed</code> value as equivalent to <code>false</code> for those <a>roles</a>.</p>
-				<p>Examples using the <code>mixed</code> value of tri-state inputs are covered in <cite><a href="https://www.w3.org/TR/wai-aria-practices-1.1/"><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices</a></cite> [[WAI-ARIA-PRACTICES-1.1]]</p>
+				<p>Examples using the <code>mixed</code> value of tri-state inputs are covered in <cite><a href="" class="practices"><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices</a></cite></p>
```

Comment:
Missing period. Add "the" for consistency?

```suggestion
				<p>Examples using the <code>mixed</code> value of tri-state inputs are covered in the <cite><a href="" class="practices"><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices</a></cite>.</p>
```